blog.html

```diff
@@ -2,24 +2,77 @@
 <html lang="en">
 	<head>
     	<meta charset="utf-8">
-    	<meta name="description" content="Your description goes here">
-    	<meta name="keywords" content="one, two, three">
+    	<meta name="description" content="Relaxr Blog">
+    	<meta name="keywords" content="Relaxr, Blog, Travel, excel, automation">
 
-		<title>Blank Template</title>
+		<title>Relaxr Blog</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>
 
 	</head>
 	<body>
 		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<h1 class="logo">Relaxr</h1>
+				<nav>
+					<ul class="topnav clearfix">
+						<li><a href="#">About</a></li>
+						<li><a href="#">FAQ</a></li>
+						<li><a href="#">Team</a></li>
+						<li><a href="#">Contact Us</a></li>	
+						<li><a href="#">Blog</a></li>	
+					</ul>
+				</nav>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+			<main class="clearfix">
+				<aside class= "clearfix">
+				<section class="categories">	
+					<ul>
+						<li><h4>Categories</h4>	
+					<ul>
+						<li>Success Stories</li>
+						<li>Stats</li>
+						<li>How Tos</li>
+						<li>Best Of </li>	
+						<li>Business Tips</li>	
+					</li>
+					</ul>
+				</section>
+				<section class="about">
+					<h4>About Relaxr</h4>
+					<span class="about">Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.</span>
+				</section>
+				<section class="ad-space">
+					<span><img src="images/ad-unit.png"></span>
+					</section>
+				</aside>
+				<article class="clearfix">
+					<h2>How I implemented Relaxr in 2 weeks and changed my life
+					</h2>
+					<p><img width="640px" src="images/blog_photo1.jpg"></p>
```

Comment:
You don't want to use inline styling here for the `img` elements. You can also assign it a `width` of 100% to have it fill up the entire width of its parent container.